tests/utils/v2Data.js

```diff
@@ -0,0 +1,27 @@
+const uuid = require('uuid');
+const { UtapiMetric } = require('../../libV2/models');
+
+function range(n, step) {
+    const vals = [...Array(n).keys()];
+    if (step) {
+        return vals.map(i => i * step);
+    }
+    return vals;
+}
+
+function makeEvent(timestamp) {
+    return new UtapiMetric({
+        timestamp,
+        uuid: uuid.v4(),
+    });
+}
+
+function generateFakeEvents(start, stop, count) {
+    const duration = stop - start;
+    const eventsEvery = duration / count;
+    return range(count, eventsEvery).map(i => makeEvent(Math.floor(start + i)));
+}
```

Comment:
Same as above, a generator here would be nice.
```suggestion
function* generateFakeEvents(start, stop, count) {
    const duration = stop - start;
    const eventsEvery = duration / count;
    for (let time of range(count, eventsEvery)) {
      yield makeEvent(Math.floor(start + time));
    }
}
```

Reply:
I agree that generators and iterators are nice, but they're currently banned by our use of `eslint-config-airbnb`. See https://github.com/airbnb/javascript#iterators--nope

Reply:
dang, however much I think functional is great and all generators are just so powerful and efficient I would easily swap over to for..of over a forEach given. Airbnb's reasoning seems to be mostly related to the web and node is probably fine but I couldn't really get much info on it.